mesonbuild/backend/ninjabackend.py

```diff
@@ -1838,20 +1844,52 @@ def scan_fortran_module_outputs(self, target):
                                        self.environment.get_build_dir())
             # Some Fortran editors save in weird encodings,
             # but all the parts we care about are in ASCII.
+# %% Fortran module dependency discovery
             with open(filename, errors='ignore') as f:
                 for line in f:
                     modmatch = modre.match(line)
+
                     if modmatch is not None:
                         modname = modmatch.group(1).lower()
-                        if modname == 'procedure':
-                            # MODULE PROCEDURE construct
-                            continue
                         if modname in module_files:
                             raise InvalidArguments(
                                 'Namespace collision: module %s defined in '
                                 'two files %s and %s.' %
                                 (modname, module_files[modname], s))
                         module_files[modname] = s
+                        continue
+                    # else:
+                    #    smodmatch = submodre.match(line)
+                       # smodrefmatch = subrefre.match(line)
```

Comment:
__[Flake8]__ [E114] indentation is not a multiple of four (comment)

<sub>posted by Sider</sub>